apps/web/pages/checkoutnew.vue

```diff
@@ -111,6 +117,8 @@
 </template>
 
 <script setup lang="ts">
+import { ref } from 'vue';
```

Comment:
Use Nuxt auto-import feature, as that's apparently one of the reasons for 'Nuxt instance unavailable' error.
That means we dont need an import file, please remove it.